### PR TITLE
Step up FIO CI using AIDE-0.18 build

### PR DESCRIPTION
**Pull request by Vincent056**
This pr is to setup CI that runs AIDE-0.18 and test AIDE-0.18 config files for File Integrity Operator.

### CODE REVIEW

**Review by rhmdnd — APPROVED** [review]
/lgtm

Glad to see this will get us started with 0.18.

File: ci-operator/config/openshift/file-integrity-operator/openshift-file-integrity-operator-master.yaml [hunk 1]
```diff
@@ -70,6 +76,23 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-aws
+- as: e2e-aws-with-aide-0.18
```
Comment (rhmdnd) [inline ci-operator/config/openshift/file-integrity-operator/openshift-file-integrity-operator-master.yaml:79]: We have the ability to say this is an optional test, right?

Comment (rhmdnd) [reply ci-operator/config/openshift/file-integrity-operator/openshift-file-integrity-operator-master.yaml:79]: I think we should be able to flag this as optional using:

```yaml
- as: e2e-aws-with-aide-0.18
  optional: true
```

I used a similar approach in https://github.com/openshift/release/pull/53502/files#diff-2d7db15a270722a31a7790ec2dbe40276c404f6b6443df19be407bcc90b5242eR82

File: ci-operator/config/openshift/file-integrity-operator/openshift-file-integrity-operator-master.yaml [hunk 2]
```diff
@@ -103,6 +109,24 @@ tests:
...
+    cluster_profile: aws
+    test:
+    - as: test
+      cli: latest
+      commands: make e2e
+      dependencies:
+      - env: IMAGE_FROM_CI
+        name: file-integrity-operator-with-aide-018
+      - env: AIDE_VERSION
+        name: 0.18
```
Comment (rhmdnd) [inline ci-operator/config/openshift/file-integrity-operator/openshift-file-integrity-operator-master.yaml:124]: I think it's failing on this, maybe we can try quoting it?

**Comment by rhmdnd** [comment]
Looks like we need to run `make jobs` to regenerate them for this change, too.

**Comment by rhmdnd** [comment]
The rehearsal failures are expected until https://github.com/openshift/file-integrity-operator/pull/550/files lands since it contains the dockerfile in question.

**Comment by Vincent056** [comment]
/pj-rehearse pull-ci-openshift-file-integrity-operator-master-e2e-aws-with-aide-018

**Comment by Vincent056** [comment]
/pj-rehearse pull-ci-openshift-file-integrity-operator-master-e2e-aws-with-aide-018

**Pull request closed without merging**